test/test.js

```diff
@@ -25,24 +25,6 @@ describe('nodejs-docker', () => {
       }, 3000);
     });
   });
-
-  it('can use a custom nodejs version', (done) => {
```

Comment:
if you want to keep the tests in travis for now, I can add a script to the travis yaml to run the structure tests in addition to what you have running now. that way we can push forward on moving these tests to the framework/not have them duplicated, but you can keep them running through your normal CI.